src/detectors/signal_analyzers/assurance.cpp

```diff
@@ -10,27 +10,15 @@
 namespace mesos {
 namespace serenity {
 
-void AssuranceDetector::shiftBasePoints() {
+void AssuranceDropAnalyzer::shiftBasePoints() {
```

Comment:
One does not contradict another. SignalDropAnalyzer analyzes some sort of signal and can signal it's drop until it have returned to normal.
From the other hand - what is "assurance" and what does it mean when it "drops"?
This name doesn't look helpful at all...